.github/workflows/pgindent.yml

```
@@ -0,0 +1,44 @@
+name: pgindent Neon
+
+on:
+  push:
+    branches:
+      - main
+      - release
+    paths:
+      - 'pgxn/**.[ch]'
+      - '.github/workflows/pgindent.yml'
+  pull_request:
+    paths:
+      - 'pgxn/**.[ch]'
+      - '.github/workflows/pgindent.yml'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.event_name == 'pull_request' }}
+
+jobs:
+  pgindent:
+    runs-on: [ self-hosted, gen3, small ]
```

We need to run workflow in a container (it has all build tools installed)

```suggestion
    runs-on: [ self-hosted, gen3, small ]
    container:
      image: 369495373322.dkr.ecr.eu-central-1.amazonaws.com/rust:pinned
      options: --init
```

---

Facepalm should be an acceptable emoji on GitHub reactions :facepalm: